### PR TITLE
adding unused NatGatways policy

### PR DESCRIPTION
**Pull request by athiruma**
1. The policy delete unused NatGatways in last 7 days
2. Search in Cloudwatch Metrics for NatGateway usages.
   1. check any value that is > 0 in the last N days. ref to: [Link](https://www.blinkops.com/blog/how-to-find-and-delete-unattached-aws-resources#:~:text=Finding%20and%20Removing%20All%20Unused%20NAT%20Gateways&text=Open%20the%20AWS%20VPC%20console,over%20the%20last%207%20days.)

### CODE REVIEW

File: cloud_governance/policy/aws/nat_gateway_unused.py [hunk 1]
```diff
@@ -8,10 +11,36 @@ class NatGatewayUnused(NonClusterZombiePolicy):
...
+    def __check_cloud_watch_logs(self, resource_id: str, days: int = 1):
+        """
+        This method returns weather the NatGateway is used in last input days
+        :param resource_id:
+        :param days:
+        :return:
+        """
+        end_time = datetime.datetime.utcnow()
+        start_time = end_time - datetime.timedelta(days=days)
+        response = self._cloudwatch.get_metric_data(start_time=start_time, end_time=end_time, resource_id=resource_id,
```
Comment (ebattat) [inline cloud_governance/policy/aws/nat_gateway_unused.py:29]: Do you think unused 1 day is enough to decide to delete it ?

File: cloud_governance/policy/aws/nat_gateway_unused.py [hunk 2]
```diff
@@ -8,10 +11,36 @@ class NatGatewayUnused(NonClusterZombiePolicy):
     to the user after 4 days and delete after 7 days.
     """
 
+    NAMESPACE = 'AWS/NATGateway'
+
     def __init__(self):
         super().__init__()
+        self._cloudwatch = CloudWatchOperations(region=self._region)
+
+    def __check_cloud_watch_logs(self, resource_id: str, days: int = 1):
```
Comment (ebattat) [inline cloud_governance/policy/aws/nat_gateway_unused.py:20]: We should create a constant:
UNUSED_DAYS  =  1

File: cloud_governance/policy/aws/nat_gateway_unused.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/policy/aws/nat_gateway_unused.py:29]: no, here Iam passing the LastUsedDay Count starting from one. If I encounter the first day is unused, I'll increment the LastUsedDay.

Comment (ebattat) [reply cloud_governance/policy/aws/nat_gateway_unused.py:30]: How much days we count till we can delete it ?

File: cloud_governance/policy/aws/nat_gateway_unused.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
+import datetime
```
Comment (ebattat) [inline cloud_governance/policy/aws/nat_gateway_unused.py:1]: I think that the policy should called: unused_nat_gateways

**Review by ebattat — COMMENTED** [review]
The description should be: 
Title: adding unused NatGatways policy
The policy delete unused NatGatways in last 7 days

**Pull request merged**